module-4/backend/week_three.md

```diff
@@ -5,15 +5,26 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+    * Node is a server side framework allowing you to write server-side Javascript. 
 2. What is Express? What is Express similar to in the Ruby world?
+    * Express is a framework that sits on top of Node that allows the creation of web applications using Node.js. It is simliar to Sinatra.
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+    * A route is created in the Server.js file that directs specific endpoints to other functions within the application
+    * app.get('/api/v1/meals', function(request, response) { Meals.getMeals(request, response) })
```

Comment:
Which can also be refactored to 

```
app.get(endpoint, Meals.getMeals)
```

JS implicitly passes arguments to callback functions